pkg/controller/datavolume-controller.go

```diff
@@ -939,6 +939,10 @@ func newPersistentVolumeClaim(dataVolume *cdiv1.DataVolume) (*corev1.PersistentV
 
 	annotations := make(map[string]string)
 
+	for k, v := range dataVolume.ObjectMeta.Annotations {
```

Comment:
This is fine, but I would suggest at least considering a PVC-Annotations section on the DV.  Passing through is certainly easy and works; but it's not always applicable and at some point you may run the risk of collisions.  I'm good either way, but I do think it should be mentioned.

Reply:
Agreed, but that would cause us to have to be vendored into kubevirt, this is a fairly minimal change that doesn't require vendoring. And yes the collisions is a concern, but we have some order guarantee, if we collide with annotations created by the controller, those annotations will win.

Reply:
Not sure about that vs an annotations key, but regardless; I'm fine with this for now.

Reply:
Ok, I guess we won't wait for CI to run

Reply:
We did wait, actually, but we have known failures that happened, I verified that those are the only failures.